### PR TITLE
Aether_Corp main page

### LINKED ISSUE #13: AETHER CORP - Initiate
### AETHER Corp Website Build

_Nuclear fuel supplier company for nuclear power plants_  **(Improvise)**

Design template pending
**To do List**

- [x] Nav bar
- [x] Footer
- [x] add some contents

### PR DESCRIPTION
**Pull request by firedrakesin**
### Main page

- Navbar done
- Search bar done
- Content done
closes #13

### CODE REVIEW

**Comment by StealthAdder** [comment]
i need to discuss

**Comment by firedrakesin** [comment]
The discussion is done.

File: Aether_Corp/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NUCLEAR FUEL</title>
+   <link href="css/style.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (StealthAdder) [inline Aether_Corp/index.html:8]: Was use of cdn based css required.

Comment (firedrakesin) [reply Aether_Corp/index.html:8]: to put search icon

Comment (StealthAdder) [reply Aether_Corp/index.html:8]: okay nice work! RESPECT!

**Pull request merged**